actions/loadconfig.py

```diff
@@ -5,22 +5,31 @@ class NapalmLoadConfig(NapalmBaseAction):
     """Load configuration into network device via NAPALM
     """
 
-    def run(self, config_file, method, **std_kwargs):
+    def run(self, config_file, method, inline_transfer, **std_kwargs):
 
-        if not method:
-            method = 'merge'
-        else:
-            method = method.lower()
-            if method not in ["merge", "replace"]:
-                raise ValueError(('{} is not a valid load method, use: '
-                                  'merge or replace').format(method))
+        try:
+            if not method:
+                method = 'merge'
+            else:
+                method = method.lower()
+                if method not in ["merge", "replace"]:
+                    raise ValueError(('{} is not a valid load method, use: '
+                                      'merge or replace').format(method))
 
-        with self.get_driver(**std_kwargs) as device:
+            with self.get_driver(**std_kwargs) as device:
+                # inline_transfer: If set it becomes True, else False
+                device.inline_transfer = inline_transfer
```

Comment:
I'm not familiar with `inline_transfer` - what is it?

Reply:
inline_transfer is specific to Netmiko. This allows the transfer of configuration within the SSH channel (i.e. CLI) as opposed to initialising an SCP session. Depending on the configuration of the router, this can fail and cause issues. I want to give users the option (in most cases to avoid forcing users to make changes).